pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -367,11 +367,9 @@ func (p *staticPolicy) GetTopologyHints(s state.State, pod *v1.Pod, container *v
 		}
 	}
 
-	// Get a list of available CPUs.
-	available := p.assignableCPUs(s)
-
 	// Get a list of reusable CPUs (e.g. CPUs reused from initContainers).
 	// It should be an empty CPUSet for a newly created pod.
+	available := p.GetAllocatableCPUs(s)
 	reusable := p.cpusToReuse[string(pod.UID)]
```

Comment:
nit: why was this change made?

Reply:
Good catch. It's an editing artifact, no reason. Will revert